src/ChipInput.js

```diff
@@ -575,6 +653,8 @@ ChipInput.propTypes = {
   chipRenderer: PropTypes.func,
   /** Whether the input value should be cleared if the `value` prop is changed. */
   clearInputValueOnChange: PropTypes.bool,
+  /** The color to render the chip based on the color palette **/
```

Comment:
The other new props, i.e. `startAdornment`, `endAdornment` and `adornmentRenderer` should also be added and documented here.